src/test/kotlin/no/nav/personbruker/dittnav/eventaggregator/beskjed/AvroBeskjedObjectMother.kt

```diff
@@ -5,11 +5,11 @@ import java.time.Instant
 
 object AvroBeskjedObjectMother {
 
-    fun createBeskjed(i: Int): Beskjed {
+    fun createBeskjed(i: Int, fodselsnummer: String): Beskjed {
```

Comment:
Vet dette blir en del steder å endre, men nevner det likevel :)
Hadde det vært en idè å droppe og måtte sende inn fødselsnummer hver eneste gang man ville bruke objectmotherne? D.v.s. beholde defaulten, og så heller endre dette i objektet man får tilbake der man har behov for det?

Reply:
Ja, det er absolutt en mulighet 😄 I fix